react/features/notifications/constants.js

```diff
@@ -3,7 +3,22 @@
 /**
  * The standard time when auto-disappearing notifications should disappear.
  */
-export const NOTIFICATION_TIMEOUT = 2500;
+export const NOTIFICATION_TIMEOUT = {
+    SHORT: 2500,
+    MEDIUM: 5000,
+    LONG: 10000,
+    STICKY: 'sticky'
```

Comment:
Where are you checking for this in order to avoid creating the timer?

Reply:
Changed STICKY value to Infinity.